extensions/pom.xml

```diff
@@ -94,6 +94,7 @@
         <module>spring-data-jpa</module>
         <module>spring-security</module>
         <module>spring-boot-properties</module>
+        <module>spring-config-server-client</module>
```

Comment:
Sounds like you need to rename the module as well ;)

Reply:
Ah.. missed it, thanks!